packages/uploadthing/src/sdk/utils.ts

```diff
@@ -96,6 +105,21 @@ export const uploadFilesInternal = async (
       });
 
       if (!s3res.ok) {
+        // tell uploadthing infra server that upload failed
+        await fetch(
+          createRequestUrl({
+            url: generateUploadThingURL("/"),
```

Comment:
probably want a pathname to this url no?

this will just create a url like `https://uploadthing.com/?actionType=failure&slug=__server-uploaded`, no?

Reply:
or are you trying to first hit the users backend which will then hit uploadthings server? seems a bit unnecessary?

Reply:
![image](https://github.com/pingdotgg/uploadthing/assets/11494384/3fab4f94-4884-4d3a-9d6d-23f4e4fdc35a)